mitodl/micromasters
# Upgraded searchkit to latest beta

#### What are the relevant tickets?
Fixes #2737 
Fixes #2151 

#### What's this PR do?
Upgrades the version of searchkit used and removes some workarounds we don't need anymore

#### How should this be manually tested?
Go to `/learners` and try out every feature on the page. It should look and act just like it did before
